crates/matrix-sdk-crypto/src/verification/sas/mod.rs

```diff
@@ -668,7 +668,9 @@ impl Sas {
     /// # futures::executor::block_on(async {
     /// # let sas: Sas = unimplemented!();
     ///
-    /// while let Some(state) = sas.changes().next().await {
+    /// let stream = sas.changes();
```

Comment:
This has to be `mut` so you can call `next` on it, right?

Reply:
Weird that `cargo test` doesn't complain, but yes.